frontend/src/app/components/tasque-board/tasque-board.component.ts

```diff
@@ -280,7 +283,10 @@ export class TasqueBoardComponent implements OnInit, OnDestroy {
   }
 
   filterTasks(): void {
-    const phrase = this.searchInput.inputValue;
+    let phrase = '';
```

Comment:
It would probably be better to just do `let phrase = this.searchInput.inputValue`. The expression `this.searchInput == false` means that searchInput contains an empty string.

Reply:
`if(this.searchInput)` checks if the input is loaded now